src/hooks/useWrapperFocus.ts

```diff
@@ -6,6 +6,52 @@ interface WrapperOptions {
   defaultFocus?: boolean;
 }
 
+const getIframeDocument = (elem: HTMLDivElement | null) => {
+  const targetIframe = Array.from(
+    document.querySelectorAll("iframe")
+  ).find((val) => val.contentDocument?.contains(elem));
+  return targetIframe && (targetIframe!.contentDocument as Document);
+};
+
+const getIframeWindow = (elem: HTMLDivElement | null) => {
+  const targetIframe = Array.from(
+    document.querySelectorAll("iframe")
+  ).find((val) => val.contentDocument?.contains(elem));
+  return targetIframe && targetIframe!.contentWindow!.window;
+};
+
+const attachEventListeners = (
+  node: HTMLDivElement | null,
+  onFocusChange: (event: any) => void
+) => {
+  const iframeWindow = getIframeWindow(node);
+  const iframeDocument = getIframeDocument(node);
```

Comment:
How about this?
```
const iframeDocument = node.ownerDocument;
const iframeWindow = iframeDocument.defaultView;
```

Reply:
I can't use either of those as they returns the global document / window not the iframe local document / window.

This is because we are setting the iframe document to be the same as the global document / window see: https://github.com/moroshko/basis/blob/19909168fe07e779ff520735fab58d9f11040281/website/src/components/ComponentPreview.js#L159